### PR TITLE
Load saved page option

### LINKED ISSUE #1163: Feature Request: (Turn off) Load last page
# Feature Request

_Load last page_

Currently this exists as a default setting, but I'd like to instead open [Main](https://play.google.com/music/listen)
If I wasn't so inexperienced with js, I'd do it myself - I doubt there'd really be much to this new settings option

### PR DESCRIPTION
**Pull request by MCManuelLP**
Default is true
Fixes #1163

### CODE REVIEW

File: src/_locales/pl-PL.json [hunk 1]
```diff
@@ -81,6 +81,8 @@
 
   "settings-option-auto-launch": "Włącz aplikację podczas uruchamiania systemu.",
 
+  "settings-option-save-page": "Automatically load last viewed page on startup",
```
Comment (MarshallOfSound) [inline src/_locales/pl-PL.json]: If you don't know the translation yourself, leave the string empty

File: src/inject/generic/core/webviewLoader.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ if (process.env.TEST_SPEC) {
 
 if (webview) {
   let once = true;
-  const targetPage = Settings.get('lastPage', 'https://play.google.com/music/listen');
+  const targetPage = Settings.get('savePage', true) ?
+Settings.get('lastPage', 'https://play.google.com/music/listen') :
```
Comment (MarshallOfSound) [inline src/inject/generic/core/webviewLoader.js]: Woo hoo, indenting

Comment (MCManuelLP) [inline src/inject/generic/core/webviewLoader.js]: Should I indent those? With 4 spaces? I wasn't sure...

Comment (MarshallOfSound) [inline src/inject/generic/core/webviewLoader.js]: As a ternary expression I would expect it to be indented 2 spaces in from the `=` sign on the top line.

File: src/_locales/pl-PL.json [hunk 3]
```diff
@@ -81,7 +81,7 @@
 
   "settings-option-auto-launch": "Włącz aplikację podczas uruchamiania systemu.",
 
-  "settings-option-save-page": "Automatically load last viewed page on startup",
+  "settings-option-scroll-lyrics": "",
```
Comment (MCManuelLP) [inline src/_locales/pl-PL.json]: aww shit >.<

**Comment by MarshallOfSound** [comment]
LGTM 👍

**Pull request merged**